### ISSUE TITLE
Time Background Color Not Being Set Properly

### ISSUE THREAD

**Issue by mmgerken** [Evilweed/protractor-beautiful-reporter#135]
The time background color in the html report is not being set correctly.  All values are being flagged with the warning and danger labels.  It appears as though the time comparison is not correct.
![asdf](https://user-images.githubusercontent.com/48369561/54061679-1ea1f400-41bf-11e9-84dd-7feed10f48ad.png)

**Comment by miller45** [comment]
I thought they were meant to be red. But no I see that there hardcoded values. Seems to be that the values 14 and 19 are on the seconds scale while the real values on millisecond scale.

**Comment by mmgerken** [comment]
I thought it might be a second vs millisecond mismatch, thanks for confirming.  

It might also be worthwhile to look at letting the user define these values instead of hard coding them.

**Comment by miller45** [comment]
@mmgerken I published the fix yesterday with version 1.2.8

**Comment by miller45** [comment]
you can configure time limits for color now with clientDefaults property:
```
// ...
       clientDefaults:{
         columnSettings:{
            warningTime: 1000,
            dangerTime: 1500
         }
      }     
// ...
```
see [customize-default-column-settings](https://github.com/Evilweed/protractor-beautiful-reporter/blob/master/README.md#customize-default-column-settings)
